service/kernel/data_management/data_conversion.cpp

```diff
@@ -31,6 +31,33 @@ namespace data_management
 {
 namespace internal
 {
+#if defined(__INTEL_COMPILER)
+template <typename T>
+static bool tryToCopyFuncAVX512(const size_t nrows, const size_t ncols, void* dst, void* ptrMin, DAAL_INT64* arrOffests)
```

Comment:
The same question here and in all occurrences below.

Reply:
done